feugy/melodie
# fix(desktop): can not release on windows

### What's in there?

Mélodie's artefact can not be built on Windows machine since we use npm@7 (comes with Node 15).
It seems that latest npm version (7.7+) are fixing the issue

Are included here:

- [x] fix(desktop): can not release on windows

### How to test?

On a windows machine, from a clean git repo:
```shell
nvm use v15.13
npm i --legacy-peer-deps
cd apps/desktop
npm run release:artifacts -- -w
```

It should build everything fine.
